Interop/Runtime/src/main/kotlin/kotlinx/cinterop/StableRef.kt

```diff
@@ -66,12 +64,11 @@ data class StableRef<out T : Any> @PublishedApi internal constructor(
     /**
      * Returns the object this handle was [created][StableRef.create] for.
      */
-    fun get() = this.ref
+    fun get() = derefStablePointer(this.stablePtr) as T
```

Comment:
It would be nice to have checked cast (as before).
I bet it is possible to return back a cast in `asStableRef`, but ignore its result.

Reply:
Not sure if I understand what do you mean, could you please hint what kind of code you'd like to see?

Reply:
`asStableRef` could be like
```
inline fun <reified T : Any> CPointer<*>.asStableRef(): StableRef<T> = StableRef<T>(this)
    .also { it.get as T }
```